package.json

```diff
@@ -66,8 +66,8 @@
     "lint:fix": "yarn lint:prettier:fix && yarn lint:eslint:fix && yarn lint:styles:fix",
     "lint:prettier": "prettier --check -- '**/*.json'",
     "lint:prettier:fix": "prettier --write -- '**/*.json'",
-    "lint:changed": "./development/get-changed-file-names.sh | grep --regexp='[.]js$' | tr '\\n' '\\0' | xargs -0 eslint",
-    "lint:changed:fix": "./development/get-changed-file-names.sh | grep --regexp='[.]js$' | tr '\\n' '\\0' | xargs -0 eslint --fix",
+    "lint:changed": "./development/get-changed-file-names.sh | grep -E --regexp='[.](js|ts|tsx)$' | tr '\\n' '\\0' | xargs -0 eslint",
+    "lint:changed:fix": "./development/get-changed-file-names.sh | grep -E --regexp='[.](js|ts|tsx)$' | tr '\\n' '\\0' | xargs -0 eslint --fix",
```

Comment:
I think the `-E` for extended regex is portable.  But if not, `\(js\|ts\|tsx\)` works too.